dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -71,37 +71,36 @@
   = f.number_field :step_speed
 
 .field.callouteditor
-  - (0...3).each do |x|
-    %label Callout #{x+1}
-
-    %div{class: "callouts#{x}"}
+  %div{id: "callout_template"}
+    %div{class: "callout_0"}
+      %label Callout
       %table
         %tr
-          %td Use callout?
+          %td Remove Callout?
           %td Element ID
           %td Localization Key
           %td Callout text
         %tr
           %td
-            %input{class: "use_callout_#{x}", type: 'checkbox'}
+            %input{class: "remove_callout id_0", type: 'checkbox'}
```

Comment:
I don't think `id_0` is needed anymore now that you're looking up ID based on `callout_0` above.